### PR TITLE
v3.0: update ansible version for v3.0

### PR DESCRIPTION
**Pull request by FateTHarlaown**
Signed-off-by: FateTHarlaown <623239185@qq.com>

### What is changed, added or deleted? 
chang the ansible version descible for v3.0

the  ansible version in doc-cn is different from requirements.txt in tidb-ansible.
### What is the related PR or file link(s)? 
no

https://pingcap.com/docs-cn/v3.0/how-to/deploy/orchestrated/ansible/
https://pingcap.com/docs-cn/v3.0/how-to/deploy/orchestrated/offline-ansible/
https://github.com/pingcap/tidb-ansible/blob/release-3.0/requirements.txt

### Which version does your change affect? 

v3.0

### CODE REVIEW

**Comment by lilin90** [comment]
@FateTHarlaown Thanks for your contribution! Please sign the CLA.

File: v3.0/how-to/deploy/orchestrated/ansible.md [hunk 1]
```diff
@@ -125,8 +125,8 @@ The key's randomart image is:
 | TiDB 版本 | tidb-ansible tag | 备注 |
 | -------- | ---------------- | --- |
 | 2.0 版本 | v2.0.10、v2.0.11 | 最新 2.0 稳定版本，可用于生产环境。 |
-| 2.1 版本 | v2.1.1 ~ v2.1.8 等 | 最新 2.1 稳定版本，可用于生产环境（建议）。 |
-| 3.0 版本 | v3.0.0-beta、v3.0.0-beta.1 等 | 目前是 beta 版本，不建议用于生产环境。 |
+| 2.1 版本 | v2.1.1 ~ v2.1.8 等 | 最新 2.1 稳定版本，可用于生产环境。 |
+| 3.0 版本 | v3.0.1 | 最新 2.1 稳定版本，可用于生产环境（建议）。 |
```
Comment (yikeke) [inline v3.0/how-to/deploy/orchestrated/ansible.md]:
```suggestion
| 3.0 版本 | v3.0.1 | 最新 3.0 稳定版本，可用于生产环境（建议）。 |
```

File: v3.0/how-to/deploy/orchestrated/ansible.md [hunk 2]
```diff
@@ -153,13 +153,13 @@ The key's randomart image is:
 
 ## 在中控机器上安装 Ansible 及其依赖
 
-以 `tidb` 用户登录中控机，请务必按以下方式通过 pip 安装 Ansible 及其相关依赖的指定版本，否则会有兼容问题。安装完成后，可通过 `ansible --version` 查看 Ansible 版本。目前 release-2.0、release-2.1 及 master 版本兼容 Ansible 2.4 及 Ansible 2.5 版本，Ansible 及相关依赖版本记录在 `tidb-ansible/requirements.txt` 文件中。
+以 `tidb` 用户登录中控机，请务必按以下方式通过 pip 安装 Ansible 及其相关依赖的指定版本，否则会有兼容问题。安装完成后，可通过 `ansible --version` 查看 Ansible 版本。目前 release-2.0与release-2.1 兼容 Ansible 2.4 版本，release-3.0与master 兼容 Ansible 2.7 版本，Ansible 及相关依赖版本记录在 `tidb-ansible/requirements.txt` 文件中。
```
Comment (yikeke) [inline v3.0/how-to/deploy/orchestrated/ansible.md]:
```suggestion
以 `tidb` 用户登录中控机，请务必按以下方式通过 pip 安装 Ansible 及其相关依赖的指定版本，否则会有兼容问题。安装完成后，可通过 `ansible --version` 查看 Ansible 版本。目前 release-2.0 与 release-2.1 兼容 Ansible 2.4 版本，release-3.0 与 master 兼容 Ansible 2.7 版本，Ansible 及相关依赖版本记录在 `tidb-ansible/requirements.txt` 文件中。
```

Comment (FateTHarlaown) [reply v3.0/how-to/deploy/orchestrated/ansible.md]: have updated.

Comment (yikeke) [reply v3.0/how-to/deploy/orchestrated/ansible.md]: Thanks~

**Comment by yikeke** [comment]
This pull request conflicted with https://github.com/pingcap/docs-cn/pull/1568. Please refer to the related changes and comments in #1568. I'll close this PR for now.

**Pull request closed without merging**